app/actors/sufia/default_admin_set_actor.rb

```diff
@@ -25,7 +25,7 @@ def ensure_admin_set_attribute!(attributes)
       end
 
       def ensure_permission_template!(admin_set_id:)
-        Sufia::PermissionTemplate.find_or_create_by!(admin_set_id: admin_set_id)
+        Sufia::PermissionTemplate.find_by(admin_set_id: admin_set_id) || create_permission_template!(admin_set_id: admin_set_id)
```

Comment:
I'm not sure we want to create an permission_template for any `admin_set_id`, only for the default one, right?  Can we do `.find_by(admin_set_id: admin_set_id) || (admin_set_id == AdminSet::DEFAULT_ID && create_default_permission_template)`